UnityProject/Assets/Scripts/Items/ServiceBell.cs

```diff
@@ -17,15 +15,16 @@ public class ServiceBell : Pickupable, IServerSpawn
 
 		[SerializeField] private SpriteHandler BellSpriteRenderer;
 
+		public bool WillInteract(HandApply interaction, NetworkSide side)
+		{
+			return interaction.Intent != Intent.Grab
+			       && interaction.Intent != Intent.Harm
+			       && interaction.TargetObject == gameObject
+			       && interaction.HandObject == null;
+		}
```

Comment:
Can you add `if (DefaultWillInteract.Default(interaction, side) == false) return false;` here, or else I think you'll be able to use the bell from any distance

Reply:
nvm, i'll do it in another PR